app/src/main/java/openfoodfacts/github/scrachx/openfood/views/splash/SplashPresenter.java

```diff
@@ -34,7 +33,7 @@ public SplashPresenter(SharedPreferences settings, ISplashPresenter.View view, C
 
     @Override
     public void refreshData() {
-        if (BuildConfig.FLAVOR.equals("off")) {
+        //if (BuildConfig.FLAVOR.equals("off")) {
```

Comment:
To be sure I understand: the idea is to load taxonomy for all "flavor" and not only for OFF ?
if yes, could you remove the commented lines ?

Reply:
Done